game-app/game-headed/build.gradle

```diff
@@ -5,6 +5,14 @@ plugins {
     id "de.undercouch.download" version "5.6.0"
 }
 
+//Activate the locked apple API with flag '--add-opens=java.desktop/com.apple.eawt.event=ALL-UNNAMED'
+application {
+    mainClass = 'org.triplea.game.client.HeadedGameRunner'
```

Comment:
I notice that the `mainClass` name is duplicated here. Do we have to duplicate it? Can we simply maybe not specify the main class name here at all? Are there other options?